charts/base-cluster/values.schema.json

```diff
@@ -1046,6 +1046,11 @@
         "enabled": {
           "type": "boolean"
         },
+        "alerting":{
+          "enabled":{
+            "type": "boolean"
```

Comment:
You added this boolean but you don't use it?

I don't get why this boolean exist in any case?